Source/Core/InputCommon/GCAdapter.cpp

```diff
@@ -28,7 +28,7 @@ static void ResetRumbleLockNeeded();
 static void Reset();
 static void Setup();
 
-static bool s_detected = false;
+static int s_status = 0;  // 0: No adapter detected; 1: Adapter detected; <0: libusb error code
```

Comment:
Any thoughts on named constants (either `enum` or `constexpr`, whichever makes more sense) for the zero/one case?

Reply:
Looks better<i>ish</i> that way. You need not comment the _possible values_ like that though (at least not solely), it would be more useful if you could comment on its _purpose_.